FirebaseSessions/Sources/Session.swift

```diff
@@ -32,13 +32,13 @@ protocol IdentifierProvider {
 }
 
 ///
-/// Identifiers is responsible for:
+/// Session is responsible for:
 ///   1) Getting the Installation ID from Installations
 ///   2) Generating the Session ID
 ///   3) Persisting and reading the Session ID from the last session
 ///   (Maybe) 4) Persisting, reading, and incrementing an increasing index
 ///
-class Identifiers: IdentifierProvider {
+class Session: SessionProtocol {
```

Comment:
As someone not familiar with all the details, the `Identifiers` name confused me because it sounds like a collection type. What is the purpose of this class? Is it to represent a "session" or just provide a caller with an installation ID?

Reply:
I think this class is supposed to be a provider of all the IDs we need, including `sessionID` and `installationID`, and is responsible for generating them and making them available.

Reply:
Thanks @leotianlizhan. Hmm, maybe then something like:
- `IdentityManager` 
- `IdentifierManager`
- `IdentityController` 
- `IdentifierController`
- `IdentityProvider`
- `IdentifierProvider`
- `IdentifierGenerator`
- `IdentifierCreator`

One of these might be a better fit? @visumickey